kernel-install/50-mkosi-initrd.install

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+export LANG=C
```

Comment:
LANG=C.UTF-8 would be better, but if this is just importing the file from `mkosi-initrd`, then it's probably better to do that in a separate step for clarity.